src/fetch/period_slippage.py

```diff
@@ -5,16 +5,43 @@
 from src.dune_analytics import DuneAnalytics, QueryParameter
 from src.file_io import File
 from src.models import Network
+from src.token_list import HOSTED_ALLOWED_BUFFER_TRADING_TOKEN_LIST_URL, get_trusted_tokens_from_url
+
+
+def generate_sql_query_for_allowed_token_list(token_list) -> str:
+    query = "allow_listed_tokens as ( Select * from (VALUES"
+    for address in token_list:
+        query = "".join([query, f"('\\{address[1:]}' :: bytea),"])
+    query = query[:-1]
+    query = "".join([query, ") AS t (token)),"])
+    return query
+
+
+def add_token_list_table_to_query(original_sub_query: str) -> str:
+    '''Inserts a the token_list table right after the WITH statement into the sql query'''
+    token_list = get_trusted_tokens_from_url(
+        HOSTED_ALLOWED_BUFFER_TRADING_TOKEN_LIST_URL)
```

Comment:
I think the constant can be called token list url with a comment that contains the long sentence you have here.